src/sage/graphs/generators/families.py

```diff
@@ -513,22 +513,16 @@ def BalancedTree(r, h):
     OUTPUT:
 
     The perfectly balanced tree of height `h \geq 1` and whose root has
-    degree `r \geq 2`. A :exc:`~networkx.exception.NetworkXError` is raised
-    if `r < 2` or `h < 1`.
-
-    ALGORITHM:
-
-    Uses the :ref:`NetworkX <spkg_networkx>` function
-    :func:`~networkx.generators.classic.balanced_tree`.
+    degree `r \geq 2`.
 
     EXAMPLES:
 
     A balanced tree whose root node has degree `r = 2`, and of height
     `h = 1`, has order 3 and size 2::
 
-        sage: G = graphs.BalancedTree(2, 1); G                                          # needs networkx
+        sage: G = graphs.BalancedTree(2, 1); G
         Balanced tree: Graph on 3 vertices
-        sage: G.order(); G.size()                                                       # needs networkx
+        sage: G.order(); G.size()
```

Comment:
Outside of the scope, but it would be good to split this doctest into 2 lines.

Reply:
ok